flexget/plugins/filter/series.py

```diff
@@ -28,7 +28,8 @@
                                             create_index)
 from flexget.utils.tools import merge_dict_from_to, parse_timedelta, parse_episode_identifier
 
-SCHEMA_VER = 13
+SCHEMA_VER = 14
+DEFAULT_SERIES_LIST_NAME = 'series_list'
```

Comment:
Default name should be "series" like with movies it's "movies"

(in hindsight I would like it to have been default in all places)